index.bs

```diff
@@ -1963,7 +1964,7 @@ i.e., VCHAR as defined in [[!RFC5234]] but without %x22 and %x5c. (Note: This me
 based on domain names MUST incorporate only LDH Labels [[!RFC5890]].) Implementations MUST match WebAuthn attestation statement
 format identifiers in a case-sensitive fashion.
 
-Attestation statement formats that may exist in multiple versions SHOULD include a version in their identifier. In effect,
+Attestation statement formats that MAY exist in multiple versions SHOULD include a version in their identifier. In effect,
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done.